tests/Http/PostTest.php

```diff
@@ -66,6 +66,9 @@ public function testCreatingAPostAndSignup()
             ->shouldReceive('userSignup')
             ->shouldReceive('userSignupPost');
 
+        // Mock the Northstar API calls.
+        $this->northstarMock;
```

Comment:
These lines aren't doing anything and can be removed (`$this->northstarMock` is a property, not a method).